src/components/Card.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import RichMarkdownEditor from 'rich-markdown-editor';
+import {connect} from 'react-redux';
+import BootstrapCard from 'react-bootstrap/Card';
+import ReactCardFlip from 'react-card-flip';
+
+class Card extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			serverRunning: true,
+			isFlipped: false
+		};
+		this.handleClick = this.handleClick.bind(this);
+		this.createBootstrapCard = this.createBootstrapCard.bind(this);
+	}
+	
+	handleClick(e) {
+		e.preventDefault();
+		this.setState(prevState => ({ isFlipped: !prevState.isFlipped }));
+	}
+	
+	createBootstrapCard(bgMode, key, textColour, json) {
+		return (
+			<BootstrapCard
+				bg={bgMode}
+				key={key}
+				text={textColour}
+				className="BootstrapCard"
+				onClick={this.handleClick}
+			>
+				<RichMarkdownEditor
```

Comment:
interesting, didn't really think of using the editor within each card, but this definitely makes sense, hopefully its not too non-lightweight

Reply:
Yeah i hope so. I figured it would be the easiest way in order to support the same block types and keep formatting consistent between the source document and the views.